govc/test/namespace.bats

```diff
@@ -113,4 +113,140 @@ load test_helper
     run govc namespace.service.info -json service2
     assert_matches DE-ACTIVATED
 
+}
+
+@test "namespace.create" {
+    vcsim_env
+
+    run govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-1
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-1 | jq)
+    assert_equal "domain-c1" $(echo $ns | jq -r '."cluster"')
+    assert_equal "0" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+
+    run govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-2 -content-libraries=lib1,lib2
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-2 | jq)
+    assert_equal "2" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+}
+
+@test "namespace.change" {
```

Comment:
what about a test that changes only associated content libraries?
and one that changes both CL and VM classes?

Reply:
also what about a namespace change on a non-existent ns?

Reply:
Added new cases for both of your suggestions